src/app/(private)/(routes)/(member)/profile/components/personal-info.tsx

```diff
@@ -248,6 +169,106 @@ export const PersonalInfo = ({ userProps }: PersonalInfoProps) => {
                   )}
                 />
               </div>
+              <div className="grid grid-cols-1 sm:grid-cols-3 items-start gap-2 sm:gap-4">
+                <FormField
+                  control={form.control}
+                  name="password"
+                  render={({ field }) => (
+                    <FormItem>
+                      <FormLabel>Password</FormLabel>
```

Comment:
For clarity and consistency with common UX patterns, consider changing the label of the first password input to either "**Current Password**" or "**Old Password**".

This would make it more intuitive for users to understand which password they should enter in this field.